R/snapshot.R

```diff
@@ -932,15 +941,26 @@ renv_snapshot_dependencies_impl <- function(project, type = NULL, dev = FALSE) {
       if (elapsed < limit)
         return()
 
+      renv_scope_options(renv.verbose = TRUE)
+
+      # tally up directories with lots of files
+      count <- count[order(count)]
+      count <- count[count >= 200]
+
       # report to user
       lines <- c(
         "",
         "NOTE: Dependency discovery took %s during snapshot.",
         "Consider using .renvignore to ignore files, or switching to explicit snapshots.",
         "See `?renv::dependencies` for more information.",
+        if (length(count)) c(
+          "",
+          sprintf("- %s: %s", format(names(count)), nplural("file", count))
+        ),
         ""
       )
 
+      renv_scope_options(renv.verbose = TRUE)
```

Comment:
I wonder if this suggests that the `writef()` should actually be a warning?

Reply:
I feel like we need something like `ewritef()`, or some other version of `writef()` that ignores `renv.verbose`.